_ml-commons-plugin/cluster-settings.md

```diff
@@ -186,3 +186,35 @@ plugins.ml_commons.native_memory_threshold: 90
 
 - Default value: 90
 - Value range: [0, 100]
+
+## Enable auto redeploy
+
+Automatically redeploys previously deployed models upon cluster failure. 
```

Comment:
We have some unsupported case like if all ML node crashed, then model state becomes `DEPLOY_FAILED`, we won't auto redeploy model with `DEPLOY_FAILED` state. We only auto redeploy `DEPLOYED` or `PARTIALLY_DEPLOYED` models